home/models.py

```diff
@@ -15,6 +15,34 @@
     TabbedInterface,
 )
 from wagtail_content_import.models import ContentImportMixin
+from wagtail.admin.forms import WagtailAdminPageForm
+
+
+class ContentPageForm(WagtailAdminPageForm):
+
+    def clean(self):
+        cleaned_data = super().clean()
+        whatsapp_body = cleaned_data.get("whatsapp_body")
+        messenger_body = cleaned_data.get("messenger_body")
+        viber_body = cleaned_data.get("viber_body")
+        total_count = 0
+        for block in whatsapp_body:
+            if block.block_type == "paragraph":
+                total_count += len(block.render())
+        if total_count >= 4096:
+            self.add_error(None, 'Whatsapp body exceeds 4096 characters')
+        total_count = 0
+        for block in messenger_body:
+            if block.block_type == "paragraph":
+                total_count += len(block.render())
+        if total_count >= 2000:
+            self.add_error(None, 'Messenger body exceeds 2000 characters')
+        for block in viber_body:
+            if block.block_type == "paragraph":
+                total_count += len(block.render())
+        if total_count >= 2000:
+            self.add_error(None, 'Viber body exceeds 7000 characters')
```

Comment:
I think either the error message or the count check here is in correct? the message says 7000 but the check is for 2000